### PR TITLE
feat: add org metadata collection

### PR DESCRIPTION
**Pull request by maxneuvians**
This PR adds a flag to the action that allows for additional queries against org data that does not need to run in every repo, but only in one one repo. The first query implemented if for users of the org.

### CODE REVIEW

**Comment by maxneuvians** [comment]
I removed NPM audit from the CI for now until we can fix the dependency issue. It is unlikely the issue will get fixed upstream - https://github.com/octokit/auth-app.js/issues/430#issuecomment-1366751991

**Review by CalvinRodo — APPROVED** [review]
I'm going to approve this, but have a question/suggestion. 

Instead of a flag would it make more sense to pass in an org/repo name and only run the check if that matches with the org/repo the action is running in?

File: src/action.js [hunk 1]
```diff
@@ -161,6 +164,32 @@ const action = async () => {
     prefix + "ActionDependencies"
   );
   console.log("✅ ActionDependencies data sent to Azure Log Analytics");
+
+  // Get central repository data
+  if (getOrgData) {
```
Comment (patheard) [inline src/action.js:169]: Does this get coerced into a boolean by `const getOrgData = core.getInput("get-org-data");`?

I'm wondering if we're going to end up here with every repo with the string `"false"` being truthy.

File: src/action.js [hunk 2]
```diff
@@ -161,6 +164,32 @@ const action = async () => {
...
+    console.log("🐿️ Getting org data");
```
Comment (patheard) [inline src/action.js:170]: :shipit:

Comment (maxneuvians) [reply src/action.js:170]: 🐿️

**Pull request merged**